torch/distributed/distributed_c10d.py

```diff
@@ -1322,6 +1322,7 @@ def init_process_group(
             pg_options=pg_options,
             timeout=timeout,
             device_id=device_id,
+            group_desc="default_pg"
```

Comment:
We don't want to expose an argument here -- the default process group's name is not for setting. No change allowed (at least as of now).

Reply:
we don't allow user to pass in a customized name in init_process_group, but this line of code is inside it and calls _new_process_group_helper to create the default P. Hence we need to specify the desc here.

Reply:
Sorry, the github review format misled me. Sounds good then.